tony-history-server/conf/application.example.conf

```diff
@@ -1,8 +1,8 @@
-play.http.secret.key="test"
+play.http.secret.key = "test"
 keytab {
-    user="abc@example.com"
-    location="/path/to/abc.keytab"
+  user = "abc@example.com"
+  location = "/path/to/abc.keytab"
 }
 tony {
-    configPath="/path/to/config/file.xml"
+  historyFolder = "/path/to/history/folder/"
```

Comment:
BTW, in my tony-conf RB, I named this property `tony.history.directory`. Can we standardize on that? Also, in the future, I think the app setup will have to be updated to read the configuration from tony-site.xml (which might be located at `$TONY_CONF_DIR/tony-site.xml`).

Reply:
With that being said, we may have to parse the `tony-site.xml` and feed it to Play server class. I can modify it in `startTHS.sh` at some point in the future.

Reply:
Yeah, just wanted to give you a heads up. We can make the change in a subsequent PR.